src/main/kotlin/coverosR3z/authentication/persistence/AuthenticationPersistence.kt

```diff
@@ -9,7 +9,7 @@ import coverosR3z.timerecording.types.EmployeeId
 
 class AuthenticationPersistence(private val pmd : PureMemoryDatabase, private val logger: ILogger) : IAuthPersistence {
 
-    override fun createUser(name: UserName, hash: Hash, salt: Salt, employeeId: EmployeeId?) : User {
+    override fun createUser(name: UserName, hash: Hash, salt: Salt, employeeId: EmployeeId) : User {
```

Comment:
all users now have an employee, it's non-nullable (good job @coveros-matt-taylor )